packages/test-utilities/etc/test-utilities.api.md

```diff
@@ -1,20 +0,0 @@
-## API Review File for "@uifabric/test-utilities"
-
-> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
-
-```ts
-
-import * as React from 'react';
-import { ReactWrapper } from 'enzyme';
-import * as renderer from 'react-test-renderer';
-
-// @public
-export function safeCreate<TProps>(content: React.ReactElement<TProps>, callback: (wrapper: renderer.ReactTestRenderer) => void): void;
-
-// @public
-export function safeMount<TComponent extends React.Component, TProps = TComponent['props'], TState = TComponent['state']>(content: React.ReactElement<TProps>, callback: (wrapper: ReactWrapper<TProps, TState, TComponent>) => void): void;

```

Comment:
Is this removal intentional? This package `@uifabric/test-utilities` has users: https://www.npmjs.com/package/@uifabric/test-utilities. I recommend keeping the file and associated script.

Reply:
The API file wasn't getting generated as part of this package's build (so the rollup probably hasn't been generated or published for awhile either). I'm not sure whether that was intentional, or whether this package really needs an API review file and rollup.

Reply:
Any update here? the PR is getting stale :)

Reply:
I ended up fixing it in api-extractor, so I'll open another PR to pick up that fix instead.